core/mutator/entry/mutation.go

```diff
@@ -15,13 +15,17 @@
 package entry
 
 import (
-	"github.com/benlaurie/objecthash/go/objecthash"
-	"github.com/golang/protobuf/proto"
 	"github.com/google/keytransparency/core/crypto/commitments"
 	"github.com/google/keytransparency/core/crypto/signatures"
 	"github.com/google/keytransparency/core/mutator"
-	tpb "github.com/google/keytransparency/core/proto/keytransparency_v1_types"
+
+	"github.com/google/trillian/crypto/keyspb"
 	"github.com/google/trillian/crypto/sigpb"
+
+	"github.com/benlaurie/objecthash/go/objecthash"
```

Comment:
did you mean to move this import?
is it the result of gofmt?

Reply:
I cleaned up the import order while I was here